argoexec/tox.ini

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+skipsdist = True
+skip_missing_interpreters = True
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}
+    PYTHONBREAKPOINT=ipdb.set_trace
+    CHARM_REPO=https://github.com/canonical/argo-operators.git
+    CHARM_BRANCH=main
+    LOCAL_CHARM_DIR=charm_repo
+
+[testenv:update-requirements]
+allowlist_externals =
+    bash
+    find
+    pip-compile
+    xargs
+commands = 
+    ; uses 'bash -c' because piping didn't work in regular tox commands
+    pip-compile requirements-sanity.in
+deps =
+    pip-tools
+description = Update requirements files by executing pip-compile on all requirements*.in files, including those in subdirs.
+
+[testenv:sanity]
+passenv = *
+allowlist_externals =
+    bash
+    tox
+    rockcraft
+deps =
+    -r requirements-sanity.txt
+commands =
+    # NOTE: .rock file should be built and available
```

Comment:
I think this note should be in a CONTRIBUTING or DEVELOPERS guide at least, so people know how this is supposed to run and what is expected to be present in the environment.

Reply:
Added top level readme.